### ISSUE TITLE
Metasploit URL Incorrect

### ISSUE THREAD

**Issue by MrJester** [fireeye/commando-vm#81]
**Describe the bug and expected behavior**
The URL being used to install Metasploit is not the correct URL.

**To Reproduce**
Install metasploit

**Screenshots**
 - metasploit.flare (exited 404) - Error while running 'C:\ProgramData\chocolatey\lib\metasploit.flare\tools\chocolateyInstall.ps1'.
 See log for details.
https://windows.metasploit.com/metasploit-framework-5.0.31%2B20190623152611-1rapid7-1-x64.msi

**Version**
 - OS: Windows 0

**Additional context**

**Comment by htnhan** [comment]
@MrJester  Thank you for reporting this issue. We have pushed a new version of metasploit. Please let us know if you still have issues.

**Comment by Milenko22** [comment]
Hi there, I'm having the same problem as MrJester. I've checked the URL for the metasploit msi and it doesn't exist. I have a handful of tools that aren't installing. I tihnk for the same reason. Is there a way to manually install the latest for the tools? When I run the installer it is obviously pulling the package versions first which then doesn't correspond in some cases. I tried editing the .nuspec file but that gets overwritten during the install. Any help?

Thanks.
